src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+abstract class Task {
+    private String taskName;
+    private boolean isCompleted;
+
+
+    public Task (String taskName, boolean isCompleted) {
+        this.taskName = taskName;
+        this.isCompleted = isCompleted;
+    }
+
+    private String outputTaskStatus () {
+        return "[" + (isCompleted ? "X" : " ") + "] ";
+    }
+
+
+    @Override
+    public String toString () {
+        return this.outputTaskStatus() + this.taskName;
+    }
+
+    public void setFinished () {
+        this.isCompleted = true;
+    }
```

Comment:
Perhaps this variable could be changed to setCompleted? It would make it clearer that this is a setter for isCompleted.